components/src/Icon/Icon.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from "prop-types";
 import styled from "styled-components";
 import { space, color } from "styled-system";
 import icons from "../../icons/icons.json";
+import theme from "../theme";
```

Comment:
I've been wondering about this one:

What's the point of using `ThemeProvider` if we import themes directly from our default theme file?
I think we should be using `ThemeConsumer` to access the theme.

Probably worth opening a ticket for this one.

Reply:
This makes sense, making a ticket:

https://nulogy-go.atlassian.net/browse/NDS-817